components/inputs/input-styles.js

```diff
@@ -20,6 +20,7 @@ export const inputStyles = css`
 		min-height: calc(2rem + 2px);
 		min-width: calc(2rem + 1em);
 		position: relative;
+		text-align: var(--d2l-input-text-align, start);
```

Comment:
This is for HTML editor?

Reply:
Color picker.
![image](https://user-images.githubusercontent.com/9042472/109063782-9670ce00-76b7-11eb-9028-ba2b39285600.png)

Reply:
For a bit more context:
![image](https://user-images.githubusercontent.com/9042472/109064004-d8017900-76b7-11eb-900d-7ddc403e23d8.png)